packages/snaps-utils/src/manifest/validation.ts

```diff
@@ -111,7 +111,7 @@ export const bip32entropy = <
 export const Bip32EntropyStruct = bip32entropy(
   type({
     path: Bip32PathStruct,
-    curve: enums(['ed25519', 'secp256k1']),
+    curve: enums(['ed25519', 'secp256k1', 'ed25519Bip32']),
```

Comment:
Can we test that this matches `SupportedCurve` somehow?

Reply:
Yes, done.